zepid/calc/utils.py

```diff
@@ -1,42 +1,89 @@
 import warnings
-import math
+from typing import NamedTuple
+
+import numpy as np
 from scipy.stats import norm
 
 
-def risk_ci(events, total, alpha=0.05, confint='wald'):
-    """Calculate two-sided (1-alpha)% Confidence interval of Risk. Note
-    relies on the Central Limit Theorem, so there must be at least 5 events
-    and 5 nonevents
+class Results(NamedTuple):
+    """
+    A simple wrapper class for the results of marginal estimates
+    """
+    point_estimate: float
+    lower_bound: float
+    upper_bound: float
+    standard_error: float
+    alpha: float
+    measure: str
```

Comment:
So this is my fault for not having Travis CI set up to check things like this, but it looks like Python 3.5 does not support variable annotations. I think this was added as part of 3.6? I would like to support both 3.5 and 3.6 if possible